src/features/preview/index.tsx

```diff
@@ -25,40 +25,44 @@ import { ClipBackground } from "../animations/ClipBackground";
 import { FourierVisualizer } from "../animations/FourierVisualizer";
 import { AutoCarouselExample } from "../animations/AutoCarousel";
 
+export const idToComponentMap: Record<DataIds, ComponentType> = {
```

Comment:
Hh, good catch @petrkonecny2 . I didn't expect the amount of animations to scale that fast. To make it clearer, could we move the ﻿`idToComponentMap` variable with the imports to a separate `preset` file, please?